src/browser/application.js

```diff
@@ -1,5 +1,5 @@
 import app from 'app';
```

Comment:
:+1: good call!

I was going for the minimal effort to get things running and missed this :blush:  I'll update the PR to also take care of the other electron imports.

thanks :muscle: